connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -57,6 +57,10 @@
     public static final String VALUE_CONVERTER_CLASS_DOC =
             "Converter class for value Connect data that implements the <code>Converter</code> interface.";
 
+    public static final String VALUE_CONVERTER_SCHEMA_CACHE_SIZE_CONFIG = "value.converter.schemas.cache.size";
```

Comment:
The intent with all the pluggable components in connect is to follow the same pattern as the clients wrt config. So we actually don't want to expose these configs via WorkerConfig. If we did, we'd basically be committing all converters to honor these configs. This cache one might make sense generally, but enable/disable schemas below definitely doesn't.